package.json

```diff
@@ -144,6 +144,7 @@
     "classnames": "~2.2.6",
     "crypto-js": "~3.3.0",
     "escape-string-regexp": "^4.0.0",
+    "i18next": "^19.8.7",
```

Comment:
This should be a dev-dependency because we need it for storybook, but it should not be required by our consumers.

The RFC shows this in more detail https://github.com/Sage/carbon/blob/master/rfcs/text/i18n.md#detailed-design